cekit/descriptor/resource.py

```diff
@@ -317,6 +325,7 @@ class _PathResource(Resource):
         'map': {
             'name': {'type': 'str', 'desc': 'Key used to identify the resource'},
             'target': {'type': 'str', 'desc': 'Target file name for the resource'},
+            'dest': {'type': 'str', 'desc': 'Destination directory inside of the container, should be used together with "target"', 'default': '/tmp/artifacts/'},
```

Comment:
the first examples in the doc don't show the usage of "target" together with "dest", or?

Reply:
Correct, `target` is optional.

Reply:
I think I could reword the description in schema. Plus I need to add this to the resource documentation, so thanks!

Reply:
This should be now resolved.